app/src/test/java/com/example/randomusergenerator/user/viewmodel/UserViewModelTest.kt

```diff
@@ -82,4 +84,16 @@ class UserViewModelTest {
         assertTrue(actualResult.isLoading)
         assertEquals(true, actualResult.isLoading)
     }
+
+    @Test
+    fun `when navigateToUserDetails is called, then verify correct function is called`() = runTest {
```

Comment:
Kindly remove the comma from the function name. I overlooked it previously but had to research it.

Reply:
okay